api/contrib/envoy/extensions/filters/network/kafka_mesh/v3alpha/kafka_mesh.proto

```diff
@@ -10,6 +10,8 @@ import "validate/validate.proto";
 option java_package = "io.envoyproxy.envoy.extensions.filters.network.kafka_mesh.v3alpha";
 option java_outer_classname = "KafkaMeshProto";
 option java_multiple_files = true;
+option go_package = "github.com/envoyproxy/go-control-plane/envoy/extensions/filters/network/kafka_mesh/v3alpha";
+option (udpa.annotations.file_status).work_in_progress = true;
```

Comment:
This seems like a duplicate of the annotation on line 16 (although this one is from the deprecated udpa namespace instead of the newer xds namespace), and it's not clear how it's related to the rest of the PR.  Is this needed?

Reply:
Hrm I'm not sure, attempting to remove it, @remyleone might remember

Reply:
I've removed it.